api/census.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/ONSdigital/dp-dataset-api/models"
+	"net/http"
+)
+
+// this is the handler for the "/census" endpoint
```

Comment:
If we are renaming to population types these function names should be updated to reflect that

Reply:
I think I have now refactored everything away from "census" nomenclature